.gitignore

```diff
@@ -1,14 +1,39 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
 *.iml
-.gradle
```

Comment:
LGTM, but I think we should keep `.gradle` ignored.

Reply:
But .gradle is still present: https://github.com/Segelzwerg/FamilyFotoAndroid/pull/20/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R38

Reply:
oh sorry my bet, thank you very much, will merge now!